pkgs/shells/bash/4.4.nix

```diff
@@ -71,10 +72,9 @@ stdenv.mkDerivation rec {
 
   # Note: Bison is needed because the patches above modify parse.y.
   depsBuildBuild = [ buildPackages.stdenv.cc ];
-  nativeBuildInputs = [ bison ]
+  nativeBuildInputs = [ bison autoconf ]
```

Comment:
Probably not a good idea to add extra dependencies for packages in stdenv. You should rather patch the configure.in (in addition to configure.ac).

Reply:
Perhaps abandon this add `-DPGRP_PIPE` into flags directly?

Reply:
(avoided requiring autoconf without having to mess with compile flags by indeed patching `configure` itself as well)